charts/example-configurations/minikube-values-renkulab-template.yaml

```diff
@@ -1,113 +1,37 @@
-#
 # Configuration file for deploying the renku chart on minikube
```

Comment:
Actually, this file and `minikube-values-gitlab.com` are now almost identical. I think it would make sense to merge the two.